### PR TITLE
APM > .NET > Logging updates for next release

### PR DESCRIPTION
**Pull request by zacharycmontoya**
### What does this PR do?
- [x] Updates references to `dotnet-tracer-` log files with `dotnet-tracer-managed-` log files
- [x] Updates .NET Framework and .NET Core configuration pages to include new `DD_TRACE_LOG_DIRECTORY` that takes precedence over `DD_TRACE_LOG_PATH` setting
- [x] Updates Tracer Startup Logs to include both log directories for Windows and Linux and includes recommendation to use the `DD_TRACE_LOG_DIRECTORY` setting to modify the location
- [x] Updates Tracer Debug Logs to recommend using setting `DD_TRACE_LOG_DIRECTORY` to modify .NET Tracer logs directory

### Motivation
We're making improvements to the logging of the .NET Tracer to make it easier to configure and understand. We're also renaming the files to avoid confusion with the continuous profiler project.

### Preview
- .NET Framework Configuration: https://docs-staging.datadoghq.com/zach.montoya/apm-dotnet-tracer-logs-update/tracing/setup_overview/setup/dotnet-framework?tab=environmentvariables#instrumentation
- .NET Core Configuration: https://docs-staging.datadoghq.com/zach.montoya/apm-dotnet-tracer-logs-update/tracing/setup_overview/setup/dotnet-core?tab=windows#instrumentation
- Tracer Startup Logs: https://docs-staging.datadoghq.com/zach.montoya/apm-dotnet-tracer-logs-update/tracing/troubleshooting/tracer_startup_logs?tab=net#file-location
- Tracer Debug Logs: https://docs-staging.datadoghq.com/zach.montoya/apm-dotnet-tracer-logs-update/tracing/troubleshooting/tracer_debug_logs?tab=net

### Additional Notes
In draft until we're release the new version of the .NET Tracer.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/troubleshooting/tracer_startup_logs.md [hunk 1]
```diff
@@ -30,11 +30,18 @@ The Java tracer does not output Diagnostics logs.  For this check, run the trace
 
 #### File Location
 
-Log files are located in `%PROGRAMDATA%\Datadog .NET Tracer\logs`.
+Logs files are saved in the following directories by default. Use the `DD_TRACE_LOG_DIRECTORY` setting to change these paths.
```
Comment (ruthnaebeck) [inline content/en/tracing/troubleshooting/tracer_startup_logs.md:33]:
```suggestion
Log files are saved in the following directories by default. Use the `DD_TRACE_LOG_DIRECTORY` setting to change these paths.
```

File: content/en/tracing/troubleshooting/tracer_startup_logs.md [hunk 2]
```diff
@@ -30,11 +30,18 @@ The Java tracer does not output Diagnostics logs.  For this check, run the trace
...
 
-- `dotnet-tracer-{processName}-{timestamp}.log` contains the Configuration log.
+| Platform | Path                                      |
+|----------|-------------------------------------------|
+| Windows  | `%ProgramData%\Datadog .NET Tracer\logs\` |
+| Linux    | `/var/log/datadog/dotnet/`                |
 
-- `dotnet-profiler.log` contains the Diagnostics logs, if any are generated.
+**Note:**: On Linux, you must create the logs directory before you enabled debug mode.
```
Comment (ruthnaebeck) [inline content/en/tracing/troubleshooting/tracer_startup_logs.md:40]:
```suggestion
**Note:**: On Linux, you must create the logs directory before you enable debug mode.
```

File: content/en/tracing/troubleshooting/tracer_startup_logs.md [hunk 3]
```diff
@@ -30,11 +30,18 @@ The Java tracer does not output Diagnostics logs.  For this check, run the trace
...
+
+- `dotnet-tracer-managed-{processName}-{timestamp}.log` contains the Configuration logs.
```
Comment (ruthnaebeck) [inline content/en/tracing/troubleshooting/tracer_startup_logs.md:42]:
```suggestion
- `dotnet-tracer-managed-{processName}-{timestamp}.log` contains the configuration logs.
```

File: content/en/tracing/troubleshooting/tracer_startup_logs.md [hunk 4]
```diff
@@ -30,11 +30,18 @@ The Java tracer does not output Diagnostics logs.  For this check, run the trace
...
+
+- `dotnet-tracer-native.log` contains the Diagnostics logs, if any are generated.
```
Comment (ruthnaebeck) [inline content/en/tracing/troubleshooting/tracer_startup_logs.md:44]:
```suggestion
- `dotnet-tracer-native.log` contains the diagnostics logs, if any are generated.
```

**Review by ruthnaebeck — CHANGES_REQUESTED** [review]
Docs review

File: content/en/tracing/troubleshooting/tracer_startup_logs.md [hunk 5]
```diff
@@ -30,11 +30,18 @@ The Java tracer does not output Diagnostics logs.  For this check, run the trace
 
 #### File Location
 
-Log files are located in `%PROGRAMDATA%\Datadog .NET Tracer\logs`.
```
Comment (andrewardito) [inline content/en/tracing/troubleshooting/tracer_startup_logs.md:33]: Only other thought here is we should maybe link back to the configuration table in either dotnet-core or dotnet-framework. Wdyt @jaycdave88 ?

**Review by andrewardito — APPROVED** [review]

**Review by ruthnaebeck — APPROVED** [review]
👍

**Pull request merged**